### PR TITLE
Dependency problem which breaks the project

### PR DESCRIPTION
**Pull request by KwintenP**
## Description

When taking a clean checkout of the repo and doing 'yarn install && ng serve', you get the following error: ERROR in Cannot read property 'getSymbolByModule' of undefined. 

### What's included?

This can be fixed by upgrading the @angular/compiler-cli to 4.0.2 which is done in the yarn.lock and package.json file.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,16 +54,16 @@
...
     "rxjs": "^5.2.0",
     "showdown": "1.6.4",
-    "zone.js": "^0.8.5",
-    "d3": "^4.6.0"
+    "zone.js": "^0.8.5"
   },
   "devDependencies": {
     "@angular/cli": "1.0.0",
-    "@angular/compiler-cli": "4.0.0",
+    "@angular/compiler-cli": "4.0.2",
```
Comment (emoralesb05) [inline package.json]: Can you change this to `^4.0.0`? So its the same as the rest of the `@angular` dependencies. 😄

**Comment by KwintenP** [comment]
Updated :)

**Comment by emoralesb05** [comment]
Thanks for the PR!

**Comment by KwintenP** [comment]
My pleasure. Thx to you guys for open sourcing this! It makes angular material usable at this moment.

**Pull request merged**